Makefile

```diff
@@ -102,6 +102,10 @@ published :
 serve : bower_components workshops/git_version.py
 	${MANAGE} runserver
 
+## serve_now    : run a server now
```

Comment:
Because I was doing development offline, and (ironically) you can only 
run a local server if you're online, because it tries to do a bower update.